apps/files_external/lib/Command/Notify.php

```diff
@@ -207,29 +213,48 @@ private function createStorage(StorageConfig $mount) {
 		return new $class($mount->getBackendOptions());
 	}
 
-	private function markParentAsOutdated($mountId, $path, OutputInterface $output) {
+	private function markParentAsOutdated($mountId, $path, OutputInterface $output, bool $verbose, bool $dryRun) {
 		$parent = ltrim(dirname($path), '/');
 		if ($parent === '.') {
 			$parent = '';
 		}
 
 		try {
-			$storageIds = $this->getStorageIds($mountId);
+			$storages = $this->getStorageIds($mountId, $parent);
 		} catch (DriverException $ex) {
 			$this->logger->logException($ex, ['message' => 'Error while trying to find correct storage ids.', 'level' => ILogger::WARN]);
 			$this->connection = $this->reconnectToDatabase($this->connection, $output);
 			$output->writeln('<info>Needed to reconnect to the database</info>');
-			$storageIds = $this->getStorageIds($mountId);
+			$storages = $this->getStorageIds($mountId, $path);
 		}
-		if (count($storageIds) === 0) {
-			throw new StorageNotAvailableException('No storages found by mount ID ' . $mountId);
+		if (count($storages) === 0) {
+			if ($verbose) {
+				$output->writeln("  no users found with access to '$parent', skipping");
```

Comment:
```suggestion
				$output->writeln("  no users found with access to '$parent', skipping", OutputInterface::VERBOSITY_VERBOSE);
```

Then verbose is not required as parameter for markParentAsOutdated.